week1/task2/Program.cs

```diff
@@ -0,0 +1,18 @@
+﻿string input = "Intellectualization";
+int vowelCount = GetVowelCount(input);
+Console.WriteLine($"Number of vowels: {vowelCount}");
+
+int GetVowelCount(string input)
+{
+    int count = 0;
+    foreach (char letter in input)
+    {
+     if(letter == 'a' ||  letter == 'e' || letter == 'i' || letter == 'o' || letter == 'u' || letter == 'A' || letter == 'E' || letter == 'I' || letter == 'O' || letter == 'U') 
```

Comment:
You can avoid having to check for both lowercase and uppercase by, for example, doing `input.ToLower()`